client/state/ui/layout-focus/test/selectors.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { getCurrentLayoutFocus, getPreviousLayoutFocus, getNextLayoutFocus } from '../../selectors';
+import layoutFocusModule from 'lib/layout-focus';
+
+let state;
```

Comment:
Minor, but I'd move this `let` declaration into the `describe` block (so it's not processed unless the tests are actually run).